utils/appliance/__init__.py

```diff
@@ -1612,7 +1612,10 @@ def configuration_details(self):
             If the data were found, it returns tuple ``(region, server name,
             server id, server zone id)``
         """
-        return db_queries.get_configuration_details(self.db)
+        try:
+            return db_queries.get_configuration_details(self.db)
+        except KeyError:
+            return None
```

Comment:
If the DB isn't there, which can happen with these migration tests, this would raise an exception. Now it correctly returns `None` when data weren't found (either because they are not there or because the DB isn't there).